edgecast/rtld/lookups/lookups_client.go

```diff
@@ -8,19 +8,20 @@ package lookups
 import (
 	"fmt"
 
-	"github.com/EdgeCast/ec-sdk-go/edgecast/client"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/internal/ecclient"
 )
 
 // New creates a new lookups API client.
-func New(c client.Client) ClientService {
-	return &Client{Client: c}
+func New(c ecclient.APIClient, cc ecclient.ClientConfig) ClientService {
+	return &Client{c, cc}
```

Comment:
Then this would be `cc.BaseAPIURL.String()` instead of just `cc`

Reply:
Actually, this is less user friendly because now they have to provide a `ClientConfig` instead of an `SDKConfig`.
It should be `func New(config edgecast.SDKConfig)`
See: https://github.com/EdgeCast/ec-sdk-go/blob/DEN-143_TestabilityImprovements/edgecast/origin/service.go

Reply:
updated